Gir.toml

```diff
@@ -2064,6 +2060,9 @@ status = "generate"
     name = "get_model"
         [object.function.return]
         nullable = false
+    [[object.function]]
+    name = "new_with_model"
+    rename = "new"
```

Comment:
Why? There's also a normal `new()` that's useful by itself and works differently

Reply:
Oh there is no `new()` and because that's how we had it in manual code before. Why?

Reply:
Before this version `gtk_tree_model_sort_new_with_model` was function in gtk_tree, so we added call for it in `TreeModelSort` for better usability, now it moved to `TreeModelSort` in upstream, so we just rename it in Rust part

Reply:
Ok :man_shrugging: